base/applications/sc/print.c

```diff
@@ -1,10 +1,10 @@
 /*
  * PROJECT:     ReactOS Services
- * LICENSE:     GPL - See COPYING in the top level directory
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
If you change the license, you are required to ask all authors of the file for their agreement.

Reply:
fine by me :)